packages/admin/admin-color-picker/src/ColorPicker.styles.ts

```diff
@@ -0,0 +1,84 @@
+import { Theme } from "@mui/material";
+import { createStyles } from "@mui/styles";
+
+import { ColorPickerProps } from "./ColorPicker";
+
+export type ColorPickerClassKey =
+    | "popperRoot"
+    | "colorPickerWrapper"
+    | "colorPalette"
+    | "colorPaletteItem"
+    | "preview"
+    | "previewIndicator"
+    | "previewIndicatorColor"
+    | "previewIndicatorEmpty"
+    | "previewIndicatorInvalid"
+    | "clearButtonWrapper";
```

Comment:
`clearButtonWrapper` is not being used, right?

```suggestion
    | "previewIndicatorInvalid";
```